javascript/javascript1/week4/palindromicSubstring.js

```diff
@@ -0,0 +1,34 @@
+var longestPalindrome = function(string) {
+
+    var length = string.length;
+    var result = "";
+  
+    var centeredPalindrome = function(left, right) {
+      while (left >= 0 && right < length && string[left] === string[right]) {
+        //expand in each direction.
+        left--;
+        right++;
+      }
+  
+      return string.slice(left + 1, right);
+    };
+  
+    for (var i = 0; i < length - 1; i++) {
```

Comment:
The code is a bit complicated, but if it works as expected, good for you! Try it on different strings and see if it makes sense.